web/src/pages/onboarding.tsx

```diff
@@ -261,6 +262,17 @@ const OnboardingPage = (props: Props): ReactElement => {
           } else {
             completeOnboarding(newProfileData, localUpdateQueue);
           }
+        } else if (querySectorId && sectorQueryParamIsValid(querySectorId)) {
+          const newProfileData: ProfileData = {
+            ...currentBusiness.profileData,
+            businessPersona: "OWNING",
+            sectorId: querySectorId,
+            operatingPhase: OperatingPhaseId.GUEST_MODE_OWNING,
```

Comment:
[dust] I don't think this is actually doing anything. I'm pretty sure `updateOperatingPhase` will get called and set this value appropriately.

Reply:
In this scenario, the user does not have an account, they are logging in as a guest user for an existing business flow. The operating phase is updated to `guest mode owning`. The `updateOperatingPhase` fn is only hit when they sign in.